### ISSUE TITLE
Mask read from .csv bug

### ISSUE THREAD

**Issue by awlostowski-noaa** [NOAA-OWP/t-route#242]
There is an apparent bug `nhd_io.read_csv()`:

``` python
def read_csv(geo_file_path, header="infer", layer_string=None):
    if geo_file_path.endswith(".zip"):
        if layer_string is None:
            raise ValueError("layer_string is needed if reading from compressed csv")
        with zipfile.ZipFile(geo_file_path, "r") as zcsv:
            with zcsv.open(layer_string) as csv:
                return pd.read_csv(csv, header=header)
    else:
        return pd.read_csv(geo_file_path)
```

the `else: return` statement needs an additional argument `header`:

```python
    else:
        return pd.read_csv(geo_file_path, header=header)
```
Otherwise, the first row of mask data is read as a header and the segment is dropped from the mask, which leads unintended network breaks.

**Comment by awlostowski-noaa** [comment]
ping @jameshalgren

**Comment by jameshalgren** [comment]
Closed by #284
